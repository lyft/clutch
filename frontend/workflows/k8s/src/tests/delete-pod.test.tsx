import React from "react";
import { shallow } from "enzyme";

import DeletePod from "../delete-pod";

describe("Terminate Instance workflow", () => {
  let component;

  beforeAll(() => {
    component = shallow(<DeletePod resolverType="clutch.k8s.v1.Pod" />);
  });

  it("renders correctly", () => {
    expect(component.debug()).toMatchSnapshot();
  });
});
