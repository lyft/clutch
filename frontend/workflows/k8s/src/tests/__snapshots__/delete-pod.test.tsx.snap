// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Terminate Instance workflow renders correctly 1`] = `
"<Wizard dataLayout={{...}} heading={[undefined]}>
  <PodIdentifier name=\\"Lookup\\" resolverType=\\"clutch.k8s.v1.Pod\\" />
  <PodDetails name=\\"Modify\\" />
  <Confirm name=\\"Confirmation\\" />
</Wizard>"
`;
